song_match/cube/lights.py

```diff
@@ -23,3 +23,36 @@ def __get_light(red: int, green: int, blue: int) -> Light:
 
 #: :class:`~cozmo.lights.Light` - Yellow light instance.
 YELLOW_LIGHT = __get_light(255, 241, 0)
+
+#: :class:`~cozmo.lights.Light` - Blue light instance.
+BLUE_LIGHT = __get_light(0, 0, 255)
+
+#: :class:`~cozmo.lights.Light` - Purple light instance.
+PURPLE_LIGHT = __get_light(150, 0, 255)
+
+#: :class:`~cozmo.lights.Light` - Green light instance.
+GREEN_LIGHT = __get_light(0, 255, 0)
+
+#: :class:`~cozmo.lights.Light` - Turquoise light instance.
+TURQUOISE_LIGHT = __get_light(0, 255, 190)
+
+#: :class:`~cozmo.lights.Light` - Salmon pink light instance.
+SALMON_PINK_LIGHT = __get_light(255, 153, 153)
+
+#: :class:`~cozmo.lights.Light` - Dark red light instance.
+DARK_RED_LIGHT = __get_light(153, 0, 0)
```

Comment:
Are all these colors nice and bright?

I think if any are dull we should just leave them out.

Reply:
Didn't test the Dark Red but I will test all of the colors again and maybe we can get distinct color for each note. We would need 6 though. I'm wondering if maybe we should just stick with the 3 brightest that way it doesn't become more about memorizing the color if/when they move onto a different randomly allocated song. I think after they play a few games and they've played the same song over and over they might learn to memorize them. We can discuss this in class